### PR TITLE
Update strings

### PR DESCRIPTION
**Pull request by Mippy**

### CODE REVIEW

**Comment by trilleplay** [comment]
this fix would mess up some currently translated strings.

File: lang/en_US.json [hunk 1]
```diff
@@ -128,57 +128,57 @@
...
   "ignored_user_removed": "I will log edits and deleted messages from `{item}` again.",
   "mod_log_channel_set": "{channel} will now be used for moderation logs.",
   "minor_log_channel_set": "{channel} will now be used for minor logging.",
   "minor_log_caching_start": "Caching recent messages for logging...",
   "no_overrides": "No overrides",
   "cog_overrides": "Cog overrides:",
-  "core_cog_no_override": "The {cog} cog is a core cog that does not allow permission overrides",
-  "cog_min_perm_violation": "The {cog} cog has a minimum permission lvl of {min_lvl} ({min_lvl_name})",
+  "core_cog_no_override": "The {cog} cog is a core cog that does not allow permission overrides.",
+  "cog_min_perm_violation": " The {cog} cog is has a minimum permission lvl of {min_lvl} ({min_lvl_name})",
```
Comment (AEnterprise) [inline lang/en_US.json]: can you remove this space at the start?

**Comment by AEnterprise** [comment]
looks good overall but changes some keys and that i need to fix and double check in the code as well so holding off until tomorow

the approved ones being reset is no problem, we can pull them from translation memory

**Comment by Mippy** [comment]
Fixed based on your review @AEnterprise

**Comment by Mippy** [comment]
Also yeah Crowdin will be messed up a little bit, but nothing that's unfixable.

File: lang/en_US.json (hunk 1 above)
Comment (Mippy) [reply lang/en_US.json]: fixed

**Pull request merged**